### ISSUE TITLE
Install fails with "Unexpected token P" on Windows Server 2012 R2

### ISSUE THREAD

**Issue by ecraig12345** [giggio/node-chromedriver#25]
I'm trying to install the package on Windows Server 2012 R2, and I get the following error. 

```
D:\chromedriver>npm i

> chromedriver@2.13.0 install D:\chromedriver
> node install.js


SyntaxError: Unexpected token P
    at Object.parse (native)
    at parseField (D:\chromedriver\node_modules\npmconf\npmconf.js:290:14)
    at D:\chromedriver\node_modules\npmconf\npmconf.js:247:15
    at Array.forEach (native)
    at Conf.add (D:\chromedriver\node_modules\npmconf\npmconf.js:246:21)
    at ConfigChain.addEnv (D:\chromedriver\node_modules\npmconf\node_modules\config-chain\i
ndex.js:207:15)
    at Conf.addEnv (D:\chromedriver\node_modules\npmconf\npmconf.js:272:30)
    at Conf.<anonymous> (D:\chromedriver\node_modules\npmconf\npmconf.js:94:10)
    at Conf.emit (events.js:95:17)
    at ConfigChain._resolve (D:\chromedriver\node_modules\npmconf\node_modules\config-chain
\index.js:281:34)

npm ERR! Windows_NT 6.2.9200
npm ERR! argv "node" "C:\\Users\\ecraig\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js"
 "i"
npm ERR! node v0.10.36
npm ERR! npm  v2.4.1
npm ERR! code ELIFECYCLE
npm ERR! chromedriver@2.13.0 install: `node install.js`
npm ERR! Exit status 8
```
`D:\chromedriver` is a directory with the code checked out from GitHub. The install also fails when node-chromedriver is requested as a dependency for another package. I tried `node-debug install.js`, but for some reason the install seems to succeed when the debugger is attached...??

**Comment by giggio** [comment]
I tried it with success on Windows 10. Can you try to clean your cache?
I need to be able to reproduce the bug to work on it. I am closing it for now, but feel free to reopen it with the steps to reproduce.

**Comment by ecraig12345** [comment]
I just updated to node.js version 0.12 and npm version 2.5.1, and now I'm FINALLY getting more details in the output:
```
undefined:1
"C:\Program Files\Sublime Text 3\subl.exe"
    ^
SyntaxError: Unexpected token P
    at Object.parse (native)
    at parseField (D:\chromedriver\node_modules\npmconf\npmconf.js:290:14)
    at D:\chromedriver\node_modules\npmconf\npmconf.js:247:15
    at Array.forEach (native)
    at Conf.add (D:\chromedriver\node_modules\npmconf\npmconf.js:246:21)
    at ConfigChain.addEnv (D:\chromedriver\node_modules\npmconf\node_modules\config-chain\index.js:207:15)
    at Conf.addEnv (D:\chromedriver\node_modules\npmconf\npmconf.js:272:30)
    at Conf.<anonymous> (D:\chromedriver\node_modules\npmconf\npmconf.js:94:10)
    at Conf.emit (events.js:107:17)
    at ConfigChain._resolve (D:\chromedriver\node_modules\npmconf\node_modules\config-chain\index.js:281:34)
```

`C:\Program Files\Sublime Text 3\subl.exe` is what I had my `EDITOR` environment variable set to. I changed the variable to use forward slashes, and everything works fine now.

**Comment by giggio** [comment]
I am not sure that this is a problem with the chromedriver package itself. The problem seems to be with `npmconf`, which is a core module to npm itself. The version of this package is fixed to 0.0.24, and it has been updated a lot since then. I will see if I can update it without breaking everything and also try to repro on Windows to make sure it fixes after the update.

**Comment by giggio** [comment]
@ecraig12345 Ok, I have updated the packages, and everything still works, but with your `EDITOR` variable it does not.
I have my `EDITOR` variable set to VIM, and the format is different from yours. I tried setting it to use sublime and it worked. Set it to (powershell):

````powershell
$env:EDITOR="'C:\Program Files\Sublime Text 3\subl.exe' *%"
````

The value should be simply: `'C:\Program Files\Sublime Text 3\subl.exe' *%`, with the single quotes around the path.
I hope it helps you.
